src/common/Header/Header.tsx

```diff
@@ -53,14 +62,29 @@ const Header = () => {
     mutateLogout();
   };
 
+  useEffect(() => {
+    sessionStorage.setItem(VOLUME_STATE_KEY, JSON.stringify(volume));
+  }, [volume]);
+
+  useEffect(() => {
+    if (audioRef.current) {
+      audioRef.current.volume = volume.effect / 100;
+      if (volume.bgm === PLAY) {
+        audioRef.current.play();
+      } else {
+        audioRef.current.pause();
+      }
+    }
```

Comment:
조건을 깔끔하게 잘 나누셨네요 ㅎㅎㅎ
하나 말씀드리자면 이렇게 early return으로 하면 depth가 작아질 것 같아요!!!
```suggestion
    if (!audioRef.current) {
      return
    }
      audioRef.current.volume = volume.effect / 100;
    if (volume.bgm === PLAY) {
        audioRef.current.play();
        return
    }
        audioRef.current.pause();
  }
```